tensorboard/webapp/metrics/views/card_renderer/BUILD

```diff
@@ -382,6 +382,7 @@ tf_ts_library(
         "//tensorboard/webapp/metrics/actions",
         "//tensorboard/webapp/metrics/data_source",
         "//tensorboard/webapp/metrics/store",
+        "//tensorboard/webapp/metrics/store:types",
```

Comment:
is this still needed?

Reply:
Yes it is needed when we import `{CardFeatureOverride}` from `'../../store/metrics_types';` in `scalar_card_test.ts`